client/src/Payment/paymentModal.js

```diff
@@ -1,19 +1,23 @@
 import React, { useEffect, useState } from 'react'
```

Comment:
@davidcandeau small note for consistency, this file should be named `PaymentModal` - not sure how that slipped past us.

Reply:
:boop: on this one, can you rename this file, please?